.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,24 @@
+---
+name: Reviewdog
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    paths:
+      - .github/workflows/golangci-lint.yml
+      - src/go/**
+
+jobs:
+  golangci-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: reviewdog/action-golangci-lint@v2
+        with:
+          golangci_lint_flags: '--config=../../.golangci.yml'
+          reporter: github-pr-review  # Comments on PR with review comments.
```

Comment:
This will not comment on forked PRs.
Use pull_request_target instead of pull_request on line 4 to give write permission to this workflow.
We just have to be careful to not use any secrets here.

Reply:
Awesome! I knew that API permissions didn't allow github-pr-review but I didn't think we had an alternative.